core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksPresenter.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.bookmark
+
+import android.util.Log
+import io.reactivex.Observable
+import io.reactivex.Scheduler
+import io.reactivex.disposables.Disposable
+import org.kiwix.kiwixmobile.core.base.BasePresenter
+import org.kiwix.kiwixmobile.core.data.DataSource
+import org.kiwix.kiwixmobile.core.di.qualifiers.Computation
+import org.kiwix.kiwixmobile.core.di.qualifiers.MainThread
+import javax.inject.Inject
+
+class BookmarksPresenter @Inject constructor(
+  private val dataSource: DataSource,
+  @param:MainThread private val mainThread: Scheduler,
```

Comment:
I don't think these need the `param:` part? I could be wrong

Reply:
Actually, it was there in history presenter so I kept it. I checked online but could not find anything about it.

Reply:
Should I remove both the params part?